NPB3.3.1/NPB3.3-OMP/sys/make.common

```diff
@@ -40,7 +40,7 @@ ${COMMON}/wtime.o: ${COMMON}/${WTIME} ../config/make.def
 	cd ${COMMON}; ${CCOMPILE} ${MACHINE} -o wtime.o ${COMMON}/${WTIME}
 # Adding ROI hooks
 ${COMMON}/hooks.o: ${COMMON}/hooks.c ../config/make.def
-	cd ${COMMON}; ${CCOMPILE} hooks.c ${GEM5DIR}/util/m5/m5op_x86.S -Wno-implicit-function-declaration -I ${GEM5DIR}/include/
```

Comment:
I think it's better this way (using the files in gem5) so we don't pollute this repo with gem5 stuff that might change. Could you explain why you think it's better to copy in the files?

Reply:
The only reason to copy the gem5 files here (instead of using GEM5DIR) is to avoid copying the gem5 source to the disk image where the NPB is going to be compiled eventually. Considering the point that gem5 utils might change in the future, I think it makes more sense to copy gem5 util/ (or entire source) to the disk image and along with the npb source before compiling.